doc/api.md

```diff
@@ -203,47 +230,32 @@ outgoing requests).
+| `lazy`                | `lazy`               | List of lazy attributes of the device                                                                                                                                                                                                                                                                                       | `[ { "name": "attr_name", "type": "Text" } ]` |
+| `commands`            | `commands`           | List of commands of the device                                                                                                                                                                                                                                                                                              | `[ { "name": "attr_name", "type": "Text" } ]` |
+| `internal_attributes` | `internalAttributes` | List of internal attributes with free format for specific IoT Agent configuration                                                                                                                                                                                                                                           | LWM2M mappings from object URIs to attributes |
+| `static_attributes`   | `staticAttributes`   | List of static attributes to append to the entity. All the updateContext requests to the CB will have this set of attributes appended.                                                                                                                                                                                      | `[ { "name": "attr_name", "type": "Text" } ]` |
+| `expressionLanguage`  | `expresionLanguage`  | optional boolean value, to set expression language used to compute expressions, possible values are: legacy or jexl. When not set or wrongly set, legacy is used as default value.                                                                                                                                          |
+| `explicitAttrs`       | `explicitAttrs`      | Boolean value to support selective ignore of measures for device so that IOTA doesn’t progress. If not specified default is `false`.                                                                                                                                                                                        | `true/false`                                  |
+| `ngsiVersion`         | `ngsiVersion`        | optional string value used in mixed mode to switch between **NGSI-v2** and **NGSI-LD** payloads. The default is `v2`. When not running in mixed mode, this field is ignored.                                                                                                                                                | `v2/ld`                                       |
```

Comment:
This change is not included in PR https://github.com/telefonicaid/iotagent-node-lib/pull/954